ts-nextjs-typeorm/README.md

```diff
@@ -18,3 +24,37 @@ You can start editing the page by modifying `src/pages/index.js`. The page auto-
 
 The `src/pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
+
+
+## Compile and Deploy with Klotho
+
+run the terminal commands:
+```sh
+# Compile the app
+npx tsc && klotho . --app ts-nextjs-typeorm -p aws
+
+# Go into the compiled directory
+cd compiled
+
+# If you didn't set the aws region as indicated in the compiler output, do that now
+pulumi config set aws:region <YOUR_REGION> -s ts-nextjs-typeorm
+
+# Set a username and password to be used for accessing `UsersDB`
+pulumi config set ts-nextjs-typeorm:usersdb_username <USERNAME> -s ts-nextjs-typeorm
+pulumi config set --secret ts-nextjs-typeorm:usersdb_password <PASSWORD> -s ts-nextjs-typeorm
+
+# npm install pulumi dependencies
+npm install
+
+# Deploy
+pulumi up -s ts-nextjs-typeorm
+
+# Outputs: {
+#   apiUrl: 'https://<...>.execute-api.<YOUR_REGION>.amazonaws.com/stage/'
+# }
+
+```
+
```

Comment:
```suggestion
## Clean Up
```sh
# Tear down when done
pulumi destroy -s ts-nextjs-typeorm
```
```

Reply:
hm format is weird, but tldr; missing clean up

Reply:
This is a special sample where i dont think we can locally test it/a little confused on this one because its meant to be a frontend. Adding cleanup

Reply:
Oh yeh I just meant that the format of my attempted suggested change was weird. not the actual readme.